lib/dfu.js

```diff
@@ -295,5 +299,54 @@ var that = module.exports = {
 		);
 	},
 
+	_udevRulesDir: '/etc/udev/rules.d/',
+
+	_missingDevicePermissions: function (stderr) {
+		return stderr && stderr.indexOf("Cannot open DFU device") >= 0;
```

Comment:
No, the full error is `dfu-util: Cannot open DFU device 2b04:d006` so it only happens when a device is plugged in but the file permissions won't allow opening it.